### ISSUE TITLE
Table structure is not correctly defined as screen reader is not associating the column headers along with the information for table cells in DetailedList Component

### ISSUE THREAD

**Issue by brajeshchandra-ms** [OfficeDev/office-ui-fabric-react#11931]
### Describe the issue:

In detailedList component Table structure is not correctly defined as screen reader is not associating the column headers along with the information for table cells. All over wherever detailsList component used, its having same issue. Also there is no props available. Hence requesting to provide provide props to associate column headers along with the information for table cells.

#### Steps to reproduce :- 

1. Navigate to detailsList component
2. Observe that screen reader is not associating the column headers along with the information for table cells

#### Actual behavior:

Table structure is not correctly defined as screen reader is not associating the column headers along with the information for table cells

#### Expected behavior:

Table structure must be defined correctly so that screen reader can associate the column headers along with the information for table cells

**Comment by xugao** [comment]
@brajeshchandra-ms - I cannot repro this issue.
Here is what I hear with Narrator + Edge when navigating to first content row in [this example](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist):
```
xsn file iconVelit esse.xsn 6/14/2017 Cillum Dolore 98 KB, non-selected, 
xsn file icon, read-only, Column Header File Type,, Row <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">501</say-as> Column <say-as interpret-as="cardinal">1</say-as> of <say-as interpret-as="cardinal">5</say-as> <mark name="ContinuousReadingEnd Id:7"/>
xsn file icon, image, <mark name="ContinuousReadingEnd Id:8"/>
read-only, Row Header Column Header Name,, Column <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">5</say-as> , Velit esse.xsn <mark name="ContinuousReadingEnd Id:9"/>
read-only, Row Header Velit esse.xsn, Column Header Date Modified,, Column <say-as interpret-as="cardinal">3</say-as> of <say-as interpret-as="cardinal">5</say-as> , 6/14/2017 <mark name="ContinuousReadingEnd Id:10"/>
read-only, Column Header Modified By,, Column <say-as interpret-as="cardinal">4</say-as> of <say-as interpret-as="cardinal">5</say-as> , Cillum Dolore <mark name="ContinuousReadingEnd Id:11"/>
read-only, Column Header File Size,, Column <say-as interpret-as="cardinal">5</say-as> of <say-as interpret-as="cardinal">5</say-as> , 98 KB <mark name="ContinuousReadingEnd Id:12"/>
```

What type of screen reader you are using and pls provide detailed repro steps

**Comment by brajeshchandra-ms** [comment]
> @brajeshchandra-ms - I cannot repro this issue.
> Here is what I hear with Narrator + Edge when navigating to first content row in [this example](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist):
> 
> ```
> xsn file iconVelit esse.xsn 6/14/2017 Cillum Dolore 98 KB, non-selected, 
> xsn file icon, read-only, Column Header File Type,, Row <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">501</say-as> Column <say-as interpret-as="cardinal">1</say-as> of <say-as interpret-as="cardinal">5</say-as> <mark name="ContinuousReadingEnd Id:7"/>
> xsn file icon, image, <mark name="ContinuousReadingEnd Id:8"/>
> read-only, Row Header Column Header Name,, Column <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">5</say-as> , Velit esse.xsn <mark name="ContinuousReadingEnd Id:9"/>
> read-only, Row Header Velit esse.xsn, Column Header Date Modified,, Column <say-as interpret-as="cardinal">3</say-as> of <say-as interpret-as="cardinal">5</say-as> , 6/14/2017 <mark name="ContinuousReadingEnd Id:10"/>
> read-only, Column Header Modified By,, Column <say-as interpret-as="cardinal">4</say-as> of <say-as interpret-as="cardinal">5</say-as> , Cillum Dolore <mark name="ContinuousReadingEnd Id:11"/>
> read-only, Column Header File Size,, Column <say-as interpret-as="cardinal">5</say-as> of <say-as interpret-as="cardinal">5</say-as> , 98 KB <mark name="ContinuousReadingEnd Id:12"/>
> ```
> 
> What type of screen reader you are using and pls provide detailed repro steps

@xugao : i am also using Narrator + Edge, Below are the steps to reroduce 

1. open detailed list grid
2. open Narrator
3. select any cell

After following above step Narrator should tell the coloumn header name along with selected cell name, For example :- if cell is having name as "office" and its header name as "product", the Narrator should tell something like "office is associated with product".

Hope this helps !! Please let me know if i can provide more information

**Comment by xugao** [comment]
@brajeshchandra-ms - have you turned on scan mode? the header info will only be announced under scan mode

**Comment by brajeshchandra-ms** [comment]
@xugao : yes i have checked with scan mode also but not getting header info, even checked the grid behaviour in office fabric detailsList https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist 
when we select any row, narrator not telling its header info(name) associated with selected cell name

**Comment by xugao** [comment]
@brajeshchandra-ms - are you using the new Edge (chromium) browser? I can repro the issue there but not old Edge browser. Pls confirm, i will log a bug on narrator side

**Comment by xugao** [comment]
bug filed on Narrator team: https://microsoft.visualstudio.com/OS/_workitems/edit/24793756

**Comment by brajeshchandra-ms** [comment]
@xugao : we are using edge browser, not new Edge (chromium) browser

**Comment by xugao** [comment]
@brajeshchandra-ms - hmm I don't repro in the old Edge though, https://github.com/OfficeDev/office-ui-fabric-react/issues/11931#issuecomment-585336827 is describing exactly what i heard (recorded using narrator buddy)

**Comment by xugao** [comment]
@brajeshchandra-ms - regardless this is outside of your control. I just checked [this bug](https://microsoft.visualstudio.com/OS/_workitems/edit/24793756) I filed to narrator team has already been fixed by their team.

**Comment by brajeshchandra-ms** [comment]
Many Thanks @xugao

**Comment by xugao** [comment]
closing this since it's external/an Edge bug.
